.github/workflows/rc-release.yml

```diff
@@ -0,0 +1,110 @@
+name: Release Candidate
+
+on:
+  push:
+    branches:
+      - 'RC-[0-9]+.[0-9]+.[0-9]+'
+    tags:
+      - 'next-*'
```

Comment:
Isn't it better to trigger with a PR to main from these branch?

Reply:
When we tag a version like 'next-1.0.2-rc-1', the publish job will start.